types/ace-modules.d.ts

```diff
@@ -890,7 +890,7 @@ declare module "ace-code/src/virtual_renderer" {
          * @param {EditSession} session The session to associate with
          **/
         setSession(session: EditSession): void;
-        session: import("ace-code/src/edit_session").EditSession;
+        session: import("ace-code").Ace.EditSession;
```

Comment:
why is this change generated?

Reply:
Seems, I've got outdated version of typescript